tests/integration/012_environment/main.tf

```diff
@@ -11,19 +11,36 @@ resource "env0_project" "test_project" {
   force_destroy = true
 }
 
+data "env0_template" "github_template_for_environment" {
+  name = "Github Integrated Template"
+}
+
 resource "env0_template" "template" {
-  name              = "Template for environment resource-${random_string.random.result}"
-  type              = "terraform"
-  repository        = "https://github.com/env0/templates"
-  path              = "misc/null-resource"
-  terraform_version = "0.15.1"
+  repository             = data.env0_template.github_template_for_environment.repository
+  github_installation_id = data.env0_template.github_template_for_environment.github_installation_id
```

Comment:
how did it work until now without installation id? is it required?

Reply:
Up until now it wasn't a github repo (even if the URL was set to github).
I needed to make it a github repo to support glob. setting 'github_installation_id' makes it a github repo.